tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
/shrug I dunno, I don't like randomness in tests since you could *potentially* miss a value that would break the tests. As for test output, I personnally don't mind long test suite since it's the CI that should be running it mostly and locally you can `.only` the part you need.